tests/validate-perfetto-proto.py

```diff
@@ -2,18 +2,22 @@
 
 import sys
 import argparse
-from perfetto.trace_processor import TraceProcessor
+from perfetto.trace_processor import TraceProcessor, TraceProcessorConfig
 
 
-def load_trace(inp, max_tries=5, retry_wait=1):
+def load_trace(inp, max_tries=5, retry_wait=1, bin_path=None):
     """Occasionally connecting to the trace processor fails with HTTP errors
     so this function tries to reduce spurious test failures"""
 
```

Comment:
```suggestion

    if bin_path and not os.path.isfile(bin_path):
        print(f"bin_path {bin_path} does not exist")
        bin_path = None
        
```
Adding some error checking.